server/sonar-qa-util/src/main/java/org/sonarqube/qa/util/pageobjects/ProjectAnalysisItem.java

```diff
@@ -34,56 +32,56 @@ public ProjectAnalysisItem(SelenideElement elt) {
   }
 
   public ProjectAnalysisItem shouldHaveEventWithText(String text) {
-    elt.find(".project-activity-events").shouldHave(text(text));
+    elt.find(".project-activity-events").shouldHave(Condition.text(text));
     return this;
   }
 
   public ProjectAnalysisItem shouldHaveDeleteButton() {
     elt.find(".js-analysis-actions").click();
-    elt.find(".js-delete-analysis").shouldBe(visible);
+    elt.find(".js-delete-analysis").shouldBe(Condition.visible);
```

Comment:
![CRITICAL](https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL') Define a constant instead of duplicating this literal ".js-delete-analysis" 3 times. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3AS1192)